lib/samples/display_ogc_api_collection/display_ogc_api_collection.dart

```diff
@@ -0,0 +1,145 @@
+                  ),
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with a basemap style and add to the map view controller.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+    _mapViewController.arcGISMap = map;
+
+    // Create an OGC feature collection table by passing in a service URL and a collection id.
+    final serviceUri = Uri.parse('https://demo.ldproxy.net/daraa');
+    const collectionId = 'TransportationGroundCrv';
+    final ogcFeatureCollectionTable =
+        OgcFeatureCollectionTable.withUriAndCollectionId(
+      uri: serviceUri,
+      collectionId: collectionId,
+    );
+    // Set the feature request mode to manual cache.
+    // In this mode, you must manually populate the table - panning and zooming won't request features automatically.
+    ogcFeatureCollectionTable.featureRequestMode =
+        FeatureRequestMode.manualCache;
+    // Load the table.
+    await ogcFeatureCollectionTable.load();
+
+    // Create a feature layer to visualize the OGC features.
+    final featureLayer =
+        FeatureLayer.withFeatureTable(ogcFeatureCollectionTable);
+    // Apply a renderer.
+    featureLayer.renderer = SimpleRenderer(
+      symbol: SimpleLineSymbol(
+        style: SimpleLineSymbolStyle.solid,
+        color: Colors.blue,
+        width: 3,
+      ),
+    );
+    // Add the feature layer to the map's operational layers.
+    map.operationalLayers.add(featureLayer);
+
+    // Re-populate the table when navigation completes e.g. after zooming or panning to a new area.
+    _mapViewController.onNavigationChanged.listen((isNavigating) async {
+      if (!isNavigating) {
+        // Set the ready state variable to false to prevent interaction while the features re-populate.
+        setState(() => _ready = false);
+        if (_mapViewController.visibleArea != null) {
+          // Create a query based on the current visible extent.
+          // Set a limit of 5000 on the number of returned features per request,
+          // because the default on some services could be as low as 10.
+          final queryParameters = QueryParameters()
+            ..geometry = _mapViewController.visibleArea!.extent
+            ..spatialRelationship = SpatialRelationship.intersects
+            ..maxFeatures = 5000;
+          // Populate the table with the query, leaving existing table entries intact.
+          // Setting outFields to null requests all fields.
```

Comment:
Should "Setting outFields to null" be "Setting outFields to an empty array"?

Reply:
Ah yep, that's a copy paste miss, thanks.